src/main/java/net/datafaker/providers/base/Aws.java

```diff
@@ -8,6 +8,7 @@
 public class Aws extends AbstractProvider<BaseProviders> {
 
     private final Text.TextRuleConfig configForRoute53ZoneId;
+    private final static String AWS_SERVICE_PREFIX = "AWS";
```

Comment:
I'm not sure where you got it from, but if I look here:

https://docs.aws.amazon.com/managedservices/latest/userguide/mq-comp.html

I can see that some services are prefixed with AWS, and some are prefixed with Amazon. (and some not prefixed at all). Did you use a different list perhaps?

Reply:
You're right. Didn't pay attention. Thanks for noticing this!

https://stackoverflow.com/a/46088871